detox/src/realms/DetoxContext.js

```diff
@@ -135,7 +135,10 @@ class DetoxContext {
    */
   async [symbols.installWorker](opts) {
     if (opts.global) {
-      opts.global['__detox__'] = this;
+      opts.global['__detox__'] = {
```

Comment:
One more interesting place.

Here I could use `this` and `new DetoxInternalsFacade(this)` instead of `require()`-ing both the facades, but, in my opinion, the latter approach is more idomatic. It just eliminates any chance that those instances could be non strict-equal.

I'd prefer to have a **total guarantee** that:

```js
require('detox') === global['__detox__'].clientApi
require('detox/internals') === global['__detox__'].internalsApi
```